lib/fixup.rb

```diff
@@ -127,14 +127,14 @@ def refresh_crew_json
   { pkg_name: 'gnome_shell', pkg_rename: nil, pkg_deprecated: true, comments: 'No longer compatible with any architecture, requires systemd.' },
   { pkg_name: 'gnome_tweaks', pkg_rename: nil, pkg_deprecated: true, comments: 'No longer compatible with any architecture, requires systemd in dep package gnome_settings_daemon.' },
   { pkg_name: 'ilmbase', pkg_rename: nil, pkg_deprecated: true, comments: 'Included in openexr.' },
-  { pkg_name: 'imagemagick6', pkg_rename: nil, pkg_deprecated: true, comments: nil },
-  { pkg_name: 'imagemagick', pkg_rename: nil, pkg_deprecated: true, comments: nil },
+  { pkg_name: 'imagemagick6', pkg_rename: nil, pkg_deprecated: true, comments: 'Obsolete version of package, no longer required.' },
+  { pkg_name: 'imagemagick', pkg_rename: nil, pkg_deprecated: true, comments: 'Metapackage made redundant by the removal of imagemagick6.' },
```

Comment:
How about making this a fake package for imagemagick7 instead?

Reply:
The imagemagick situation is a little complicated because of the way the metapackage could have installed things-- I think my original plan was to eventually rename imagemagick7 to imagemagick, but this is outside of the scope of this PR regardless.